pkgs/applications/display-managers/sddm/default.nix

```diff
@@ -69,7 +68,7 @@ in
 stdenv.mkDerivation {
   name = "sddm-${version}";
 
-  nativeBuildInputs = [ lndir makeQtWrapper ];
+  nativeBuildInputs = [ extra-cmake-modules lndir makeQtWrapper ];
```

Comment:
Is this really needed? This derivation is a wrapper so it shouldn't need any CMake-related dependencies...